### ISSUE TITLE
Multi GPU batch normalization on MXNet

### ISSUE THREAD

**Issue by liuzhengzhe** [apache/incubator-mxnet#5888]
When we train a model with batch norm layer with multi gpus using MXNet, will the mean and variance swap among the multi gpus? or only calculated within one GPU?

Thanks

**Comment by alanhuang1990** [comment]
Each gPU has its mean and variance,  and gamma and beta will be shared among all gpus.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!

**Comment by zhanghang1989** [comment]
MXNet Gluon Cross-GPU Batch Norm is Implemented here 
 https://github.com/zhanghang1989/MXNet-Gluon-SyncBN. Feel free to try and leave comments.

**Comment by sammieghabra** [comment]
@zhanghang1989 in the current BatchNorm, is running_mean and running_var synchronized across all GPUs? What about in the SyncBatchNorm?
